go/vumitools/billing_worker.py

```diff
@@ -171,8 +175,9 @@ def process_inbound(self, config, msg, connector_name):
         log.debug("Processing inbound: %r" % (msg,))
         msg_mdh = self.get_metadata_helper(msg)
         try:
-            yield self.create_transaction_for_inbound(msg)
-            msg_mdh.set_paid()
+            if not self.disable_billing:
+                yield self.create_transaction_for_inbound(msg)
+                msg_mdh.set_paid()
```

Comment:
Should we maybe log that we're not billing for a message because it's been disabled?